src/App.js

```diff
@@ -1,12 +1,198 @@
 import React from 'react';
 import './App.css';
+import Counter from './components/Counter/Counter';
+import Button from './components/Button/Button';
+import { useForm } from "react-hook-form";
 
 function App() {
-  return (
-    <>
-      <h1>Fruitmand bezorgservice</h1>
-    </>
-  );
+
+
+    const [count, setCount] = React.useState({
+        strawberries: 0,
+        bananas: 0,
+        apples: 0,
+        kiwis: 0
+    });
```

Comment:
Ik zou voor elk fruitsoort een aparte useState gebruiken. Het werkt goed zo hoor, maar nu zal elke update van de `count` een onnodige rerender veroorzaken voor het fruit wat niet verlaagd of opgehoogd wordt